README.md

```diff
@@ -9,6 +9,9 @@ with excellent error reporting. You can use it to process complex data or
 computer languages and build transformers, interpreters, compilers and other
 tools easily.
 
+Peggy is the successor of [PEG.js](https://github.com/pegjs/pegjs) which had been abandoned by its maintainer.
+Peggy version 1.x.x is backwards compatible with the most recent PEG.js release. Switching from PEG.js to Peggy should be as simple as uninstalling `pegjs` and installing `peggy` instead.
```

Comment:
... and changing your `require`

Reply:
and any scripts which use the `pegjs` binary...